src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -3666,6 +3672,13 @@ await using (stream)
 
         private async Task<TcpConnectionHeaderMessage.SupportedFeatures> NegotiateProtocolVersionAsyncForCluster(string url, TcpConnectionInfo info, Stream stream, JsonOperationContext ctx, string tag)
         {
+            bool compressionSupport = false;
+#if NETCOREAPP3_1_OR_GREATER
```

Comment:
can be removed? we are running on .NET 7 so this is alwas true and this is in Raven.Server, not Raven.Client?

Reply:
you're right.
it's only relevant to the client, not to the server.
It was removed.